sql/telemetry/fxa_users_daily_v1/query.sql

```diff
@@ -38,11 +36,9 @@ CREATE TEMP FUNCTION udf_contains_tier1_country(
       )
   )
 );
-  --
```

Comment:
This statement looks like it was generated by `prepend_udf_usage_definitions`.

Reply:
yeah, it was